src/Controller/ModulesController.php

```diff
@@ -173,6 +173,165 @@ public function view($id) : ?Response
         return null;
     }
 
+    /**
+     * Display new translation form.
+     *
+     * @param string|int $id Resource ID.
+     * @return \Cake\Http\Response|null
+     */
+    public function createTranslation($id) : ?Response
```

Comment:
Why not use a `TranslationsController` instead? 
`ModulesController` is becoming too big...

Reply:
Good point, I was thinking the same... :soon: